osu.Game/Online/HubClientConnector.cs

```diff
@@ -19,6 +19,9 @@ public class HubClientConnector : PersistentEndpointClientConnector, IHubClientC
     {
         public const string SERVER_SHUTDOWN_MESSAGE = "Server is shutting down.";
 
+        public const string VERSION_HASH_HEADER = @"OsuVersionHash";
```

Comment:
Any reason for not making this `X-Osu-Version-Hash`? I think the `X-` prefix is pretty standard for custom headers.

Reply:
Mostly not wanting to faff about with

- sending this header under 2 header names in the interim
- updating the server side to read the new one
- waiting a bit so that the old header name can be deleted from client

If you're fine with said faffery (or a hard break) I can go do it.

Reply:
I'd just send under both for now and we can figure what to do about it in the future. Remove old at soem point.

Reply:
See also conversation at https://discord.com/channels/188630481301012481/188630652340404224/1263134581679198310 I guess.